src/lib/mina_base/parties_logic.ml

```diff
@@ -1,3 +1,5 @@
+(* parties_logic.ml *)
```

Comment:
Still not a fan of this style :)

Reply:
This style comes from a textbook I used years ago. It's helpful if you ever print out a file. I guess people don't do that so much these days.